.github/workflows/builder-dispatch.yml

```diff
@@ -16,10 +16,8 @@ on:
         type: string
         # please remember to update build-packages.yml as well
         default: >-
-          el-7
           el-8
```

Comment:
should this list be in sync with the list in `builder.yml`?

Reply:
no, in fact these days perhaps it should explicitly not overlap with it at all I think. `builder-dispatch` should contain a good default set for shipping packages (even if we don't builder-dispatch for releases any more). `builder` is older than all the more specific ones, and was for daily testing of distros we -don't- autobuild for in any other way. 

builder-dispatch should be in sync with builder-packages (or ideally the list would live in one of them, but that's tricky). `builder` (which we should maybe give a useful suffix to explain what it does) should then cover distros we do not, or do not yet, ship for